.gitattributes

```diff
@@ -12,29 +12,6 @@
 ###############################################################################
 *.cs     diff=csharp
 
-###############################################################################
-# Set the merge driver for project and solution files
-#
-# Merging from the command prompt will add diff markers to the files if there
-# are conflicts (Merging from VS is not affected by the settings below, in VS
-# the diff markers are never inserted). Diff markers may cause the following 
-# file extensions to fail to load in VS. An alternative would be to treat
-# these files as binary and thus will always conflict and require user
-# intervention with every merge. To do so, just uncomment the entries below
-###############################################################################
-*.sln       merge=binary
```

Comment:
Instead of deleting this, we could also simply comment them out so that future readers will know that they have the option to return to binary merge driver.

Reply:
My preference is to just delete.